HomeWorks/HW1/BurrowsWheeler/BurrowsWheeler.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Linq;
+
+namespace BWT;
+
+public static class BurrowsWheelerTransform
+{
+    public static (string transformed, int index) Transform(string input)
```

Comment:
```suggestion
    public static (string Transformed, int Index) Transform(string input)
```
Ага, именовать элементы кортежей надо, но с большой буквы